scripts/build/termux_setup_self_debug.sh

```diff
@@ -0,0 +1,15 @@
+termux_print_stacktrace() {
+	local func src lineno
+	echo "💥 Error occurred at line $1 in script ${BASH_SOURCE[1]}"
+	echo "🔍 Stack trace:"
+	for i in "${!FUNCNAME[@]}"; do
+		func="${FUNCNAME[$i]}"
+		src="${BASH_SOURCE[$i]}"
+		lineno="${BASH_LINENO[$((i - 1))]}"
+		# skip print_stacktrace itself
```

Comment:
That comment is gonna need another revision.
What are we actually skipping here, and under what condition?

Reply:
It is needed to not include termux_print_stacktrace itself (to not confuse anyone).